File: java/404.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode(int x) { val = x; }
 * }
 */
class Solution {
    int sum = 0;
    public int sumOfLeftLeaves(TreeNode root) {
        findLeft(root);
        return sum;
    }
    public void findLeft(TreeNode node){
        if(node == null){
            return ;
        }
        if(node.left != null){
            if(node.left.left == null &&node.left.right == null){
                sum += node.left.val;
            }
            else{
                findLeft(node.left);
            }
        }
        findLeft(node.right);
    }
}

File: go/404.go
/**
 * Definition for a binary tree node.
 * type TreeNode struct {
 *     Val int
 *     Left *TreeNode
 *     Right *TreeNode
 * }
 */

func sumOfLeftLeaves(root *TreeNode) int {
	if root == nil {
		return 0
	}

	left := sumOfLeftLeaves(root.Left)
	right := sumOfLeftLeaves(root.Right)
	if root.Left != nil && root.Left.Left == nil && root.Left.Right == nil {
		left += root.Left.Val
	}
	return left + right
}